client/src/pages/admin/users/UserModal.tsx

```diff
@@ -101,6 +110,26 @@ export default function UserModal({ selectedUserID, onClose }: UserModalProps) {
     });
   };
 
+  const handleDeleteUserClicked = () => {
+      let fName = getUserResult.data.user.firstName;
+      let lName = getUserResult.data.user.lastName;
+      let privilege = getUserResult.data.user.privilege;
+      if (privilege == Privilege.ADMIN) {
+          let result = window.confirm(
+              `Are you sure you wish to delete ${fName} ${lName}'s account? This cannot be undone.`
+          );
+          if (result) {
+              deleteUser({
+                  variables: {userID: getUserResult.data.user.id},
+                  refetchQueries: [{query: GET_USER, variables: {id: selectedUserID}}],
+              });
+          };
+      }
+      else {
+          window.alert("You don't have the privilege level to delete users!")
+      }
+  }
```

Comment:
Are you running Prettier? This file should be formatted with 2 spaces automatically for you. Perhaps we can look into it together in our next meeting.